mne/channels/channels.py

```diff
@@ -414,14 +414,12 @@ def inst_has(attr):
 class InterpolationMixin(object):
     """Mixin class for Raw, Evoked, Epochs
     """
-    def interpolate_bads_eeg(self):
-        """Interpolate bad channels
 
-        Operates in place.
-        """
+    def interpolate_bads(self):
+        """Interpolate bad channels"""
         from .interpolation import _interpolate_bads_eeg
-        _interpolate_bads_eeg(self)
-        return self
+        from ..forward import _interpolate_bads_meg
+        return _interpolate_bads_meg(_interpolate_bads_eeg(self.copy()))
```

Comment:
I think it makes more sense to have one `_interpolate_bads` function that triages the set of channels to interpolate. Imagine if we end up having more channel types, this gets unwieldy quickly. And it's less DRY if we end up wanting to do interpolation in more than one place.